src/libraries/Common/src/System/Runtime/InteropServices/ArrayMarshaller.cs

```diff
@@ -1,6 +1,8 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+#nullable enable
```

Comment:
What necessitated this?

Reply:
System.Diagnostics.PerformanceCounter does not appear to have it enabled, so I added this to avoid the warning about using nullable annotations in a context without it not enabled.

Reply:
Can we instead add `<Nullable>annotations</Nullable>` to the .csproj?

Reply:
Ah, thanks - I completely forgot about `annotations`. I only tried `enabled` (and undid it after the resulting wall of errors).